src/lwc/geFormField/geFormField.js

```diff
@@ -137,6 +142,13 @@ export default class GeFormField extends LightningElement {
             return event.detail.checked.toString();
         } else if(this.isRichText) {
             return event.target.value;
+        } else if(this.isLookup && event.detail.value) {
+            const val = event.detail.value;
+            if(typeof val === 'string') {
+                return val;
+            } else if(Array.isArray(val)) {
+                return val[0] ? val[0] : null;
```

Comment:
Usually it's not ideal to return null in a function/method. Do we have any concerns with returning null here?

Reply:
I don't think so for this. We use null to represent blank / empty values for lookups when resetting them. If there is no value, I think coalescing it to null is ok.